api-module/src/main/java/com/romanticpipe/reviewcanvas/domain/shopadmin/application/usecase/ShopAdminUseCaseImpl.java

```diff
@@ -79,4 +88,25 @@ public void deleteShopAdmin(Integer adminId, AdminRole adminRole, LocalDateTime
 		AdminAuth adminAuth = adminAuthService.findByAdminId(adminId, adminRole);
 		adminAuth.logout();
 	}
+
+	@Override
+	@Transactional(readOnly = true)
+	public PageResponse<GetShopInfoResponse> getShopInfos(PageableRequest pageable) {
+		PageResponse<ShopAdmin> shopAdmins = shopAdminService.findAll(pageable);
+
+		return shopAdmins.map(shopAdmin -> this.convertToGetShopInfoResponse(shopAdmin));
```

Comment:
```suggestion
		return shopAdmins.map(shopAdmin -> {
		    return GetShopInfoResponse.builder()
			.mallId(shopAdmin.getMallId())
			.createdAt(shopAdmin.getCreatedAt())
			.mallName(shopAdmin.getMallName())
			.mallNumber(shopAdmin.getMallNumber())
			.approveStatus(shopAdmin.isApproveStatus())
			.reviewsAmount(reviewService.countByShopAdminId(shopAdmin.getId()))
			.reviewLayoutDesign(
				reviewLayoutService.validateByShopAdminId(shopAdmin.getId()).getReviewLayoutDesign())
			.build();
		});
```
메서드로 빼지 말고 합치는건 어떤가요? 재사용 가능성이 없어보여 한 눈에 볼 수 있도록 하는 것도 좋을거 같습니다!